### ISSUE TITLE
Why is there a dependancy on jquery?

### ISSUE THREAD

**Issue by AlainPilon** [mozilla/metrics-graphics#716]
The library is great, but there is no way I will add jquery to my app just to use it... Also, on the NPM sites, is says the only dependency is D3.

**Comment by almossawi** [comment]
D3 is the only dependency. jQuery enables tooltips, but MG works without it.

**Comment by AlainPilon** [comment]
well, I dont use the tooltips and I was getting an error here: https://github.com/mozilla/metrics-graphics/blob/master/src/js/common/chart_title.js#L31

**Comment by davideschiera** [comment]
True, but you get there only if tooltips are enabled: https://github.com/mozilla/metrics-graphics/blob/367d035d6c18439f95eca2ced4b549b300414905/src/js/common/chart_title.js#L23-L31.

Here is the property you might want to set to `false` https://github.com/mozilla/metrics-graphics/wiki/Graphic#show_tooltips (in the code: https://github.com/mozilla/metrics-graphics/blob/7b2f88189da8fcba055a4e9af75fe19b837144eb/src/js/common/data_graphic.js#L115).

**Comment by AlainPilon** [comment]
May I suggest that we either put the default setting to `false` or based it on the presence of jquery? jQuery usage is not as widespread as what it used to be.

**Comment by hamilton** [comment]
We'll be working on removing the dependency on jquery / bootstrap soon for the 2.14 release. This lib was written in a different era entirely.
